docs/organisation/engineering-team.md

```diff
@@ -0,0 +1,194 @@
+the reviewers by solving the merge conflicts to speed up the process).
+- Pull Requests should only be merged if at least two contributors reviewed and approved it.
+
+## Scope of Issues and Commits
+- Stay focused on the issue and its specifics: Sometimes it is tempting to do more changes in a pull request and simply 
+add a nice little feature after mentioning it in the issue tracker. Please do not do this. Contributors will look at the 
+title of issues usually to check if it is relevant to them. For example, if an issue is about changing a font, do not also 
+change the color even if this seems like a small change to you. Many projects have a design template and standard colors 
+etc. that they want to achieve. So your changes might need to be discussed in a bigger setting even if they seem small to
+you. The same applies to many other areas.
+- Do only the changes in a pull request that are mentioned in the issue. Do not change anything else without ever 
+mentioning it (remember match issues with pull requests).
+
+## Branch Policies
+
+Most SEF Projects have:
+- A development branch (which is the working branch. Please commit to this branch.)
+- A master branch (which is the stable branch). Some projects also keep additional branches.
+- Pull requests should be made to the default branch.
+
+## Documentation
+
+- Please ensure you provide information about your code and the functioning of your app as part of every code contribution.
+- Add information on technologies and the code into the code itself and into documentation files e.g. into the Readme.MD 
+or /docs folder. Documentation is not a thing that should be done at the end after a few weeks or months of coding! 
+Documentation is a continuous effort at every step of software development.
+- If you add new libraries or technologies add information about them into the Readme file.
+- If you implement a different design or change the CSS in a web project, please add information to the design 
+specifications/template. If there are no other design specifications in the project yet, why not start them and add a 
+section to the Readme.MD to start with?
+- Always help to keep documentation up to date and make it easy for new contributors to join the project.
+
+## Developer Hierarchy
+
+Our contributors are granted badges with each milestone. These achievements can be added to your linkedin profiles, and 
+social media profiles. For further details refer [here](https://sef.discourse.group/t/granting-badges-to-developers/351)
+The hierarchy of badges and the requirements are as follows:
+
+- Level 0 - Contributor Kraken 
+  - Has made the first pull request
+  - Has introduced themselves on SEF Hive
+- Level 1 - Titan Developer
+  - Has made three pull requests
+  - Has level 0 badge
+- Level 2 - Git Gojira
+  - Has made five pull requests
+  - Has level 1 badge
+- Level 3 - SEF Prodigy
+  - Has passed the quiz
```

Comment:
Shall we add the quiz link (https://forms.gle/sp8XESD6h5HdQJ5M9) here? @anjula-sack

Reply:
yeah we need to.

Reply:
okay I'll make an amend. Everything else seems to be okay?